tests/resources/test_user.py

```diff
@@ -16,3 +44,100 @@ def test_user_collection_creation():
     session = FakeSession()
```

Comment:
Why not use the pytest fixture here?

Reply:
Oh IDK I didn't write this line xD
Git Blame points at Justin as the author, he might remember!